wcomponents-core/src/main/java/com/github/bordertech/wcomponents/WTextArea.java

```diff
@@ -65,6 +69,30 @@ protected TextAreaModel getComponentModel() {
 	}
 
 	/**

```

Comment:
No problem. I built this to sanitize input from a WTextArea in rich text mode and this is the lowest impact time to do that level of sanitization (it also means only cleaned HTML goes into storage).

I suppose I shouldn't trust HTML coming from a data source any more than I should trust it coming from a "human".

Maybe we have to sanitize both ways for WTextArea: incoming if richText & editable and outgoing for all richText?